automatic/bleachbit.install/update.ps1

```diff
@@ -0,0 +1,45 @@
+import-module au
+
+$releases = 'https://www.bleachbit.org/download/windows'
+$betas = 'https://download.bleachbit.org/beta/'
+
+function global:au_GetLatest {
+    $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
+    $filename = ($download_page.links | ? href -match '.exe$' | select -First 1 -expand href).Replace('/download/file/t?file=','')
+    $version = $filename -split '-' | select -First 1 -Skip 1
+
+    $page_betas    = Invoke-WebRequest -Uri $betas -UseBasicParsing
+    $path_beta     = $betas + ($page_betas.links | select -Last 1 -expand href)
+    $page_beta     = Invoke-WebRequest -Uri $path_beta -UseBasicParsing
+    $filename_beta = $page_beta.links | ? href -match 'setup.exe$' | select -First 1 -expand href
+    $version_beta  = $filename_beta -split '-' | select -First 1 -Skip 1
+
+    if ([version]$version_beta -gt [version]$version) {
```

Comment:
Please make use of streams, instead of just picking either 'beta' or 'stable' release.
See one of the [existing packages][streams] for inspiration

[streams]: https://github.com/chocolatey/chocolatey-coreteampackages/search?l=PowerShell&q=streams&type=&utf8=%E2%9C%93 "Streams Search"

Reply:
Streams are for non linear releases. Beta versions in most cases are just another linear version.

Reply:
I suggested @dimqua  to use this method that I used with [nssm](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/automatic/nssm/update.ps1) package. `nssm` package releases are linear, i.e.`v1 v2 v3b1 v3b2 v3 v31 v4...`

Reply:
so you're telling me that they never releases a hotfix for the latest stable release, while they have a beta version in the works?
If so, then yes it would be a linear release, otherwise it wouldn't

Reply:
The latest releases are ```1.10, 1.11.0-beta, 1.11.2-beta, 1.11.4-beta, 1.12, 1.13-alpha, 1.15-alpha, 1.17-beta```.

Reply:
@AdmiringWorm, majority of the tools are like that. Only major mainstream tools such as programming languages have long term support for releases. 

Even if it happens that vendor releases hotfix while newer beta is present, its exception rather then a rule.